### ISSUE TITLE
Uncaught Error: Cannot access empty property

### ISSUE THREAD

**Issue by xrat** [mizunashi-mana/dokuwiki-plugin-mdpage#64]
**This is a bug report.**

| Name | Value |
| --- | --- |
| DokuWiki Version | 2020-07-29 |
| PHP Version | 7.0.33-0+deb9u9 |
| OS | Debian 9 |
| Web Server | Apache 2.4 |
| mdpage Version | 2020-09-13 |

Details:

```
error GET /dw/doku.php HTTP/2.0 mod_proxy_fcgi.c(769) AH01071: Got error 'PHP message: 
PHP Fatal error:  Uncaught Error: Cannot access empty property in /home/dw/lib/plugins/mdpage/syntax.php:85
Stack trace:
#0 /home/dw/lib/plugins/mdpage/syntax.php(65): syntax_plugin_mdpage->renderWithRenderer(Object(Doku_Renderer_metadata), '\\nText\\n', Array)
#1 /home/dw/inc/parser/renderer.php(117): syntax_plugin_mdpage->render('metadata', Object(Doku_Renderer_metadata), Array)
#2 /home/dw/inc/parserutils.php(516): Doku_Renderer->plugin('mdpage', Array, 3, '\\nText\\n')
#3 /home/dw/inc/parserutils.php(282): p_render_metadata('tmp:mdpagetest', Array)
#4 /home/dw/inc/common.php(244): p_get_metadata('tmp:mdpagetest')
#5 /home/dw/doku.php(93): pageinfo()
#6 {main}\n  thrown in /home/dw/lib/plugins/mdpage/syntax.php on line 85
```
I am at a loss here. I replaced the DW markdownextra plugin with mdpage and now get this error on any page that includes `<markdown>`. I tried the setting github-flavored instead of markdown-extra but the error was the same.

`mdpagetest` is
```
Line 1

<markdown>
Text
</markdown>
```

**Comment by xrat** [comment]
Works with **mdpage 2020-06-14**.

**Comment by mizunashi-mana** [comment]
@xrat Thanks for the report.

Perhaps, this is caused by PHP 7.0.  I tested with PHP 7.3 when I released 2020-09-13, then I cannot find any troubles.  However, I added some ill syntax for PHP 7.0, sorry.

I hope to fix it by #65 .  The fix will be released soon.

**Comment by mizunashi-mana** [comment]
I just released #65.  Please update.

**Comment by xrat** [comment]
Updated and pages are loading fine now! Thanks a lot!
